src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+//todo: modularise code
 public class Duke {
+    public static final int MAX_COMMANDS = 100;
+
     public static void main(String[] args) {
```

Comment:
The main method has too many if-else statements. Consider breaking it into different methods so that it is shorter and easier to debug if necessary.